pyvista/examples/downloads.py

```diff
@@ -5140,11 +5957,16 @@ def download_cells_nd(load=True):  # pragma: no cover
     >>> dataset = examples.download_cells_nd()
     >>> dataset.plot(cpos="xy")
 
+    .. seealso::
+
+        :ref:`CellsNd Dataset <cells_nd_dataset>`
```

Comment:
This should say "Cells Nd" with a space but codespell will complain about this, and adding "Nd" to the list of allowed words in `accept.txt` doesn't work. Adding "nd" does work, but then this will cause codespell to miss future typos of "and" or "2nd". This can't be ignored inline either since it's part of a docstring.

So, I think removing the space here as a workaround is a better alternative than adding "nd" to the codespell dictionary.